LocalRun error : Container startup failed

**Describe the bug**
 org.testcontainers.containers.ContainerLaunchException: Container startup failed when trying to runLocal cloudflow application

**To Reproduce**
It reproduce with my own project and in taxi-ride cloudflow example.

**Expected behavior**
Run application without errors

**Additional context**
`[error] org.testcontainers.containers.ContainerLaunchException: Container startup failed
[error] at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:327)
[error] at org.testcontainers.containers.KafkaContainer.doStart(KafkaContainer.java:94)
[error] at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:308)
[error] at cloudflow.sbt.CloudflowLocalRunnerPlugin$.setupKafka(CloudflowLocalRunnerPlugin.scala:220)
[error] at cloudflow.sbt.CloudflowLocalRunnerPlugin$.$anonfun$projectSettings$19(CloudflowLocalRunnerPlugin.scala:136)
[error] at scala.Option.getOrElse(Option.scala:189)
[error] at cloudflow.sbt.CloudflowLocalRunnerPlugin$.$anonfun$projectSettings$11(CloudflowLocalRunnerPlugin.scala:136)
[error] at cloudflow.sbt.CloudflowLocalRunnerPlugin$.$anonfun$projectSettings$11$adapted(CloudflowLocalRunnerPlugin.scala:88)
[error] at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error] at sbt.std.Transform$$anon$4.work(Transform.scala:67)
[error] at sbt.Execute.$anonfun$submit$2(Execute.scala:281)
[error] at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:19)
[error] at sbt.Execute.work(Execute.scala:290)
[error] at sbt.Execute.$anonfun$submit$1(Execute.scala:281)
[error] at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
[error] at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
[error] at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error] at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error] at java.lang.Thread.run(Thread.java:748)
[error] Caused by: org.testcontainers.containers.ContainerFetchException: Can't get Docker image: RemoteDockerImage(imageName=confluentinc/cp-kafka:5.4.3, imagePullPolicy=DefaultPullPolicy())
[error] at org.testcontainers.containers.GenericContainer.getDockerImageName(GenericContainer.java:1278)
[error] at org.testcontainers.containers.GenericContainer.logger(GenericContainer.java:612)
[error] at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:317)
[error] at org.testcontainers.containers.KafkaContainer.doStart(KafkaContainer.java:94)
[error] at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:308)
[error] at cloudflow.sbt.CloudflowLocalRunnerPlugin$.setupKafka(CloudflowLocalRunnerPlugin.scala:220)
[error] at cloudflow.sbt.CloudflowLocalRunnerPlugin$.$anonfun$projectSettings$19(CloudflowLocalRunnerPlugin.scala:136)
[error] at scala.Option.getOrElse(Option.scala:189)
[error] at cloudflow.sbt.CloudflowLocalRunnerPlugin$.$anonfun$projectSettings$11(CloudflowLocalRunnerPlugin.scala:136)
[error] at cloudflow.sbt.CloudflowLocalRunnerPlugin$.$anonfun$projectSettings$11$adapted(CloudflowLocalRunnerPlugin.scala:88)
[error] at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error] at sbt.std.Transform$$anon$4.work(Transform.scala:67)
[error] at sbt.Execute.$anonfun$submit$2(Execute.scala:281)
[error] at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:19)
[error] at sbt.Execute.work(Execute.scala:290)
[error] at sbt.Execute.$anonfun$submit$1(Execute.scala:281)
[error] at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
[error] at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
[error] at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error] at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error] at java.lang.Thread.run(Thread.java:748)
[error] Caused by: com.github.dockerjava.api.exception.NotFoundException: Status 404: {"message":"No such image: testcontainers/ryuk:0.3.0"}
[error]
[error] at org.testcontainers.shaded.com.github.dockerjava.core.DefaultInvocationBuilder.execute(DefaultInvocationBuilder.java:241)
[error] at org.testcontainers.shaded.com.github.dockerjava.core.DefaultInvocationBuilder.post(DefaultInvocationBuilder.java:125)
[error] at org.testcontainers.shaded.com.github.dockerjava.core.exec.CreateContainerCmdExec.execute(CreateContainerCmdExec.java:33)
[error] at org.testcontainers.shaded.com.github.dockerjava.core.exec.CreateContainerCmdExec.execute(CreateContainerCmdExec.java:13)
[error] at org.testcontainers.shaded.com.github.dockerjava.core.exec.AbstrSyncDockerCmdExec.exec(AbstrSyncDockerCmdExec.java:21)
[error] at org.testcontainers.shaded.com.github.dockerjava.core.command.AbstrDockerCmd.exec(AbstrDockerCmd.java:35)
[error] at org.testcontainers.shaded.com.github.dockerjava.core.command.CreateContainerCmdImpl.exec(CreateContainerCmdImpl.java:595)
[error] at org.testcontainers.utility.ResourceReaper.start(ResourceReaper.java:91)
[error] at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:203)
[error] at org.testcontainers.LazyDockerClient.getDockerClient(LazyDockerClient.java:14)
[error] at org.testcontainers.LazyDockerClient.listImagesCmd(LazyDockerClient.java:12)
[error] at org.testcontainers.images.LocalImagesCache.maybeInitCache(LocalImagesCache.java:68)
[error] at org.testcontainers.images.LocalImagesCache.get(LocalImagesCache.java:32)
[error] at org.testcontainers.images.AbstractImagePullPolicy.shouldPull(AbstractImagePullPolicy.java:18)
[error] at org.testcontainers.images.RemoteDockerImage.resolve(RemoteDockerImage.java:66)
[error] at org.testcontainers.images.RemoteDockerImage.resolve(RemoteDockerImage.java:27)
[error] at org.testcontainers.utility.LazyFuture.getResolvedValue(LazyFuture.java:17)
[error] at org.testcontainers.utility.LazyFuture.get(LazyFuture.java:39)
[error] at org.testcontainers.containers.GenericContainer.getDockerImageName(GenericContainer.java:1276)
[error] at org.testcontainers.containers.GenericContainer.logger(GenericContainer.java:612)
[error] at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:317)
[error] at org.testcontainers.containers.KafkaContainer.doStart(KafkaContainer.java:94)
[error] at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:308)
[error] at cloudflow.sbt.CloudflowLocalRunnerPlugin$.setupKafka(CloudflowLocalRunnerPlugin.scala:220)
[error] at cloudflow.sbt.CloudflowLocalRunnerPlugin$.$anonfun$projectSettings$19(CloudflowLocalRunnerPlugin.scala:136)
[error] at scala.Option.getOrElse(Option.scala:189)
[error] at cloudflow.sbt.CloudflowLocalRunnerPlugin$.$anonfun$projectSettings$11(CloudflowLocalRunnerPlugin.scala:136)
[error] at cloudflow.sbt.CloudflowLocalRunnerPlugin$.$anonfun$projectSettings$11$adapted(CloudflowLocalRunnerPlugin.scala:88)
[error] at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error] at sbt.std.Transform$$anon$4.work(Transform.scala:67)
[error] at sbt.Execute.$anonfun$submit$2(Execute.scala:281)
[error] at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:19)
[error] at sbt.Execute.work(Execute.scala:290)
[error] at sbt.Execute.$anonfun$submit$1(Execute.scala:281)
[error] at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
[error] at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
[error] at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error] at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error] at java.lang.Thread.run(Thread.java:748)
[error] (my-project / Test / runLocal) org.testcontainers.containers.ContainerLaunchException: Container startup failed`

--- a/core/project/Dependencies.scala
+++ b/core/project/Dependencies.scala
@@ -13,7 +13,7 @@ object Version {
   val Spark         = "2.4.5"
   val Flink         = "1.10.0"
   val KafkaClients  = "2.5.0"
-  val TestcontainersKafka = "1.15.0" 
+  val TestcontainersKafka = "1.15.1" 
 }
 
 object Library {

